http_requests.py

```diff
@@ -75,41 +77,71 @@ def mount_default_adapter(
     return session
 
 
+def _create_logging_queue():
+    '''Set up a logging thread and return the queue used for communicating with it.
+    '''
+    config_set_name = 'internal_active'
+    els_index_name = 'github_access_stacktrace'
+    logging_queue = queue.SimpleQueue()
+    els_logging_client = _create_logging_client(config_set_name)
+    thread = threading.Thread(
```

Comment:
queue and thread ought to be module attrs (either safeguard the queue with a lock, or initialise it immediately)